experimenter/experimenter/nimbus-ui/src/components/PageSummary/index.test.tsx

```diff
@@ -977,7 +977,7 @@ describe("PageSummary", () => {
     });
     render(<Subject mocks={[mockRollout]} />);
     expect(
-      screen.queryByTestId("desktop-min-version-warning"),
+      screen.queryByTestId("warning-desktop-min-version"),
     ).not.toBeInTheDocument();
```

Comment:
Semantically, this should be `expect(...).toBeNull()`. `expect(node).not.toBeInTheDocument()` should be used if `node` is an actual DOM element. ie, you would use this with an element you grabbed earlier to check if it got removed.

querySelector et al return `null` when they dont find an element, so expecting `null` not to be in the document is semantically wrong because it never will be. We should expect the method to not find anything instead.